pytorch_lightning/core/memory.py

```diff
@@ -183,7 +183,9 @@ def __init__(self, model, mode: str = MODE_DEFAULT):
         self._mode = mode
         self._layer_summary = self.summarize()
         # 1 byte -> 8 bits
-        self._precision_megabytes = (self._model.precision / 8.0) * 1e-6
+        # TODO: how do we compute precisin_megabytes in case of mixed precision?
```

Comment:
Isn't this TODO solved?

Reply:
No, but it was handled false before. So currently it matches old (but probably wrong) behavior by assuming mixed precision has 16bit only weights